File: CS/CS.NET-VB.NET/ASP.NET 4.0 Book/Source/Chapter 11/VB/LINQtoObjectVB/App_Code/Customer.vb
Imports Microsoft.VisualBasic
Public Class Customer
    Private _CustID As Integer
    Public Property CustID() As Integer
        Get
            Return _CustID
        End Get
        Set(ByVal value As Integer)
            _CustID = value
        End Set
    End Property
    Private _CustName As String
    Public Property CustName() As String
        Get
            Return _CustName
        End Get
        Set(ByVal value As String)
            _CustName = value
        End Set
    End Property
    Private _CustAge As Integer
    Public Property CustAge() As Integer
        Get
            Return _CustAge
        End Get
        Set(ByVal value As Integer)
            _CustAge = value
        End Set
    End Property
	Public Sub New(ByVal intCustID As Integer, ByVal strCustName As String, ByVal intCustAge As Integer)
        _CustID = intCustID
        _CustName = strCustName
        _CustAge = intCustAge
    End Sub
End Class

File: CS/CS.NET-VB.NET/ASP.NET 4.0 Book/Source/Chapter 11/CS/LINQtoObjectCS/App_Code/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

/// <summary>
/// Summary description for Customer
/// </summary>
public class Customer
{
    private int _CustID;
    public int CustID
    {
        get { return _CustID; }
        set { _CustID = value; }
    }
    private string _CustName;
    public string CustName
    {
        get { return _CustName; }
        set { _CustName = value; }
    }
    private int _CustAge;
    public int CustAge
    {
        get { return _CustAge; }
        set { _CustAge = value; }
    }
    public Customer(int intCustID, string strCustName, int intCustAge)
    {
        _CustID = intCustID;
        _CustName = strCustName;
        _CustAge = intCustAge;
    }

}